src/aat/resources/features/F-037/S-577.td.json

```diff
@@ -0,0 +1,122 @@
+{
+  "title": "must return successfully all and only the respective fields with READ access for a Case Worker and a Solicitor",
+  "_extends_": "S-577_Update_Case_For_Case_Worker_Data_Base",
+  "_guid_": "S-577",
+  "specs": [
+    "an active Solicitor profile in CCD with a specific variation of ACLs on a case type",
+    "an active Solicitor profile having full permissions on the same case type",
+    "prompts an update to DocumentField2, made by the privileged user who just created the case",
+    "contains updated values for DocumentField2"
+  ],
+  "user": {
+    "username": "befta.caseworker.2.solicitor.2@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "body": {
+      "data": {
+        "DocumentField2":{
+          "document_url":"http://dm-store:8080/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
```

Comment:
there is some url validation on the preview build which will reject this document_url. The domain needs to be changed from `dm-store:8080` to `dm-store-aat.service.core-compute-aat.internal` (so this url should be updated to `http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943`). Please can you update it here and all other occurrences in the PR

Reply:
Thanks, thats updated.